package.json

```diff
@@ -59,21 +59,21 @@
     "eslint-config-standard": "^16.0.0"
   },
   "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": ">=4.0.1",
```

Comment:
Was this moved there automatically?

Reply:
Yes when I ran `npm install`.

Reply:
That's an unrelated change, then. And it should be made prior to any other change made with npm. As a separate PR. No apologies from me on this. This is how I roll.

Reply:
Well, others people, when they'll do PRs will probably change that too I guess.
But fine like that, we'll see, I updated, so it's doesn't change these lines. :+1: